include/pybind11/numpy.h

```diff
@@ -767,6 +769,9 @@ template <typename T, int ExtraFlags = array::forcecast> class array_t : public
     explicit array_t(ShapeContainer shape, const T *ptr = nullptr, handle base = handle())
         : array(std::move(shape), ptr, base) { }
 
+    explicit array_t(size_t count, const T *ptr = nullptr, handle base = handle())
```

Comment:
I was thinking of that. This is a new constructor from the merge with the master branch. I wasn't sure what to do with it. If `count` is `ssize_t`, do we need a test for non-negativity?